src/mono/mono/mini/aot-compiler.c

```diff
@@ -10708,6 +10708,18 @@ execute_system (const char * command)
 
 #ifdef ENABLE_LLVM
 
+#ifdef TARGET_WIN32
+#define OPT_NAME "opt.exe"
+#else
+#define OPT_NAME "opt"
+#endif
+
+#ifdef TARGET_WIN32
+#define LLC_NAME "llc.exe"
+#else
+#define LLC_NAME "llc"
+#endif
```

Comment:
should be `HOST_WIN32`, I think?

AOT compiler targets are like "Android", "Wasm", etc, I think?

Reply:
That file actually uses TARGET_* for most of the platform tooling, clang, gcc, as, ld etc, so using this change just followed along. I would say that parts of this is host and parts of this is target, since some platforms might use different tooling depending on TARGET, but I agree that just the file extension is part of host.